src/codegen/sys/statics.rs

```diff
@@ -52,7 +47,7 @@ pub fn only_for_glib(w: &mut Write) -> Result<()> {
         "pub type gpointer = *mut c_void;",
         "",
         "#[repr(C)]",
-        "#[derive(Debug)]",
+        "#[derive(Copy,Clone)]",
```

Comment:
`"#[derive(Copy, Clone)]",` ?

Reply:
Union containing volatile field need to be copyable as well.
Alternatively we could stop generating unions with volatile fields.

Reply:
Sorry that it was unclear: You just forgot space between.

Reply:
Please make 2 last fixes change current commits.